src/Symfony/Bundle/FrameworkBundle/Resources/config/session.xml

```diff
@@ -5,7 +5,7 @@
     xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
 
     <parameters>
-        <parameter key="session.metadata.storage_key">_sf2_meta</parameter>
+        <parameter key="session.metadata.storage_key">_sf3_meta</parameter>
```

Comment:
This is a BC break for projects relying on the meta (as existing sessions use the old key)